config/tomcat.yml

```diff
@@ -39,3 +39,6 @@ redis_store:
   database: 0
   timeout: 2000
   connection_pool_size: 2
+geode_store:
+  version: 0.+
+  repository_root: "https://tomcat-javabuildpack-spike.s3.amazonaws.com/geode-store"
```

Comment:
Yeah, it'll need to be a public repository with a permissive license for us to host it.  This goes back to the fact that the CFF needs to take a special exception for US export compliance that can only be satisfied by 100% open source code.